### PR TITLE
Add Grouple Websites

### PR DESCRIPTION
**Pull request by MikeZeDev**

### CODE REVIEW

File: web/src/engine/websites/AllHentai.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Tags } from '../Tags';
+import icon from './AllHentai.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as Grouple from './decorators/Grouple';
+//import { Text } from '../SettingsManager';
+//import { WebsiteResourceKey as R } from '../../i18n/ILocale';
+
+// TODO: add url setting
```
Comment (ronny1982) [inline web/src/engine/websites/AllHentai.ts:9]: Can't the URL be updated during`Initialization`?

File: web/src/engine/websites/MintManga_e2e.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        id: 'mintmanga',
+        title: 'MintManga'
+    },
+    container: {
+        url: 'https://2.mintmanga.one/blagoslovenie_nebojitelei__A3b0f6',
+        id: '/blagoslovenie_nebojitelei__A3b0f6',
+        title: 'Благословение небожителей',
+    }, /* Need to be logged, and chapter link change with user id anyway
+    child: {
+        id: '/blagoslovenie_nebojitelei__A3b0f67/vol8/98',
```
Comment (ronny1982) [inline web/src/engine/websites/MintManga_e2e.ts:14]: Parameter `?mtr=1` is missing

File: web/src/engine/websites/AllHentai_e2e.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    container: {
+        url: 'https://20.allhen.online/nochiu_ona_nimfomanka',
+        id: '/nochiu_ona_nimfomanka',
+        title: 'Ночью она нимфоманка'
+    },
+/*   Need to be logged, and chapter link change with user id anyway
+
+    child: {
+        id: '/nochiu_ona_nimfomanka/vol1/4',
```
Comment (ronny1982) [inline web/src/engine/websites/AllHentai_e2e.ts:16]: Parameter `?mtr=1` is missing

**Review by ronny1982 — APPROVED** [review]
Consider adding `Grouple_e2e.ts` including all websites which are using this template, for faster verification when changes are made to the template.

File: web/src/engine/websites/AllHentai_e2e.ts (hunk 3 above)
Comment (MikeZeDev) [reply web/src/engine/websites/AllHentai_e2e.ts:16]: It doesnt matter, as the test will NEVER be good. On this website chapter will ALWAYS has user token as parameter and there is none when not logged. Fine, i'll add it

File: web/src/engine/websites/MintManga_e2e.ts (hunk 2 above)
Comment (MikeZeDev) [reply web/src/engine/websites/MintManga_e2e.ts:14]: Same reasoning than above.

**Pull request merged**